inclusion_connect/users/admin.py

```diff
@@ -1,10 +1,25 @@
 from django.contrib import admin
-from django.contrib.auth import admin as auth_admin
+from django.contrib.auth import admin as auth_admin, forms as auth_forms
 
 from .models import User
 
 
+class AdminPasswordChange(auth_forms.AdminPasswordChangeForm):
+    def save(self, commit=True):
+        self.user.must_reset_password = True
+        return super().save(commit)
+
+
 @admin.register(User)
 class UserAdmin(auth_admin.UserAdmin):
     model = User
     readonly_fields = ["username"]
+
+    change_password_form = AdminPasswordChange
+
+    fieldsets = auth_admin.UserAdmin.fieldsets + (
+        (
+            "Informations",
+            {"fields": ("must_reset_password",)},
+        ),
+    )
```

Comment:
On peut le remonter dans le premier `fieldset` avec le changement suivant :
```suggestion
    def get_fieldsets(self, request, obj=None):
        fieldsets = super().get_fieldsets(request, obj)
        assert fieldsets[0] == (None, {"fields": ("username", "password")})
        new_fieldsets = copy.deepcopy(fieldsets)
        new_fieldsets[0][1]["fields"] = ["username", "password", "must_reset_password"]
        return new_fieldsets
```